packages/p2p/src/components/modal-manager/modals/ad-error-tooltip-modal/ad-error-tooltip-modal.jsx

```diff
@@ -4,26 +4,127 @@ import { observer } from 'mobx-react-lite';
+                            currency: account_currency,
+                        }}
+                    />
+                );
+            case 'advert_min_limit':
+                return (
+                    <Localize i18n_default_text='This ad is not listed on Buy/Sell because its maximum order is lower than the minimum amount you can specify for orders in your ads.' />
+                );
+            case 'advert_remaining':
+                return (
                     <Localize
-                        i18n_default_text='Your ad is not listed on Buy/Sell because its minimum order is higher than your Deriv P2P available balance ({{balance}} {{currency}}).'
+                        i18n_default_text='This ad is not listed on Buy/Sell because its minimum order is higher than the ad’s remaining amount ({{remaining_amount}} {{currency}}).'
+                        values={{
+                            remaining_amount,
+                            currency: account_currency,
+                        }}
+                    />
+                );
+            case 'advertiser_ads_paused':
+                return (
+                    <Localize i18n_default_text='This ad is not listed on Buy/Sell because you have paused all your ads.' />
+                );
+            case 'advertiser_balance':
+                return (
+                    <Localize
+                        i18n_default_text='This ad is not listed on Buy/Sell because its minimum order is higher than your Deriv P2P available balance ({{balance}} {{currency}}).'
                         values={{
                             balance: general_store.advertiser_info.balance_available,
-                            currency: my_ads_store.advert_details?.account_currency,
+                            currency: account_currency,
                         }}
                     />
+                );
+            case 'advertiser_daily_limit':
+                return (
+                    <Localize
+                        i18n_default_text='This ad is not listed on Buy/Sell because its minimum order is higher than your remaining daily limit ({{remaining_limit}} {{currency}}).'
+                        values={{
+                            remaining_limit: advert_type === buy_sell.BUY ? daily_buy_limit : daily_sell_limit,
+                            currency: account_currency,
+                        }}
+                    />
+                );
+            case 'advertiser_temp_ban':
+                return (
+                    <Localize i18n_default_text='You’re not allowed to use Deriv P2P to advertise. Please contact us via live chat for more information.' />
+                );
+            default:
+                return <Localize i18n_default_text='Your ad is not listed' />;
+        }
+    };
+
+    const getMultipleErrorMessages = error_statuses =>
+        error_statuses.map((status, index) => (
+            <div key={index}>
```

Comment:
Can we use any other key instead of index?

Reply:
As discussed, since there are no other unique values, index will be used as key